src/gui/qjacktrip.cpp

```diff
@@ -281,7 +286,7 @@ QJackTrip::QJackTrip(int argc, QWidget* parent)
 
     // One of our arguments will always be --gui, so if that's the only one
     // then we don't need to show the warning message.
-    if ((!gVerboseFlag && m_argc > 2) || m_argc > 3) {
+    if (((!gVerboseFlag && m_argc > 2) || m_argc > 3) && !launchedByDeeplink) {
```

Comment:
Stuff like this is why you need to be constantly testing NOVS builds as well. (No need to fix this one, I've already done it, and it'll ultimately land in a PR to clean up main and create better separation between core and vs code.)

Reply:
But the URL scheme should be generic. Isn't it?

Reply:
As far as I'm aware, it's not. But that's not the issue here. It's the reliance on a variable that does not exist when NO_VS is set. (And the same issue happens a number of times in main, which also needs a clean up in general. I'm hoping to encapsulate off some of the new stuff when I get a chance, but I at least now have a version which will build properly - will upload later in the week when I'm not on the road and have time to test properly.)